### ISSUE TITLE
Evaluated variable hover values buggy for very long values (>100,000 characters)

### ISSUE THREAD

**Issue by harrisont** [harrisont/fastbuild-vscode#38]
Example: `.ProjectBuildCommand` in `T:\p4\Patcher\Build\MakeRunnableProjectConfig.bff`
![image](https://user-images.githubusercontent.com/144260/230676163-2d987f0a-b571-449b-a61d-5d99e4ce1ae2.png)

I'm guessing that VS Code is truncating the result.

**Comment by harrisont** [comment]
Repro case:
```
.Chars000100 = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789'
.Chars001000 = '$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$'
.Chars010000 = '$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$'
.Chars099981 = '$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$012345678901234567890123456789012345678901234567890123456789012345678901234567890'
.Chars099982 = '$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars010000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars001000$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$$Chars000100$0123456789012345678901234567890123456789012345678901234567890123456789012345678901'
```
`.Chars099981` is 99981 characters and is not truncated.
`.Chars099982` is 99982 characters and is not truncated.

**Comment by harrisont** [comment]
Released in [v0.5.9](https://github.com/harrisont/fastbuild-vscode/releases/tag/v0.5.9).
